package.json

```diff
@@ -1,54 +1,31 @@
 {
-  "name": "thimble",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.4.0",
-    "@types/node": "^16.11.19",
-    "@types/react": "^17.0.38",
-    "@types/react-dom": "^17.0.11",
-    "classnames": "^2.3.1",
-    "express": "^4.17.2",
-    "mini-css-extract-plugin": "2.4.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^6.2.1",
-    "react-scripts": "5.0.0",
-    "typescript": "^4.5.4",
-    "web-vitals": "^2.1.3"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "dev": "nodemon server.js"
-  },
-  "engines": {
-    "node": "16.13.0"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "devDependencies": {
-    "nodemon": "^2.0.15"
-  }
-}
+   "name": "thimble",
+   "version": "1.0.0",
+   "private": "true",
+   "description": "",
+   "main": "server.js",
+   "scripts": {
+      "start": "node server/server.js",
+      "dev": "nodemon server/server.js",
+      "build": "cd app && npm install && npm run build"
+   },
+   "repository": {
+      "type": "git",
+      "url": "git+https://github.com/HBoots/thimble.git"
+   },
+   "author": "",
+   "license": "ISC",
+   "engines": {
+      "node": "16.13.0"
+   },
+   "bugs": {
+      "url": "https://github.com/HBoots/thimble/issues"
+   },
+   "homepage": "https://powerful-crag-58139.herokuapp.com",
+   "devDependencies": {
+      "nodemon": "^2.0.15"
```

Comment:
Hi Jeff, you found my comment! Wow, you did a lot of work :)    I notice that  "prettier": "2.5.1" seems to have disappeared from devDependencies. If i understand what you've done, I believe it should still go here, in this file?